tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -407,7 +424,7 @@ EOF
     # bsc#1210906
     assert_script_run("sysctl -w vm.unprivileged_userfaultfd=1");
     # Installing Whereabouts plugin
-    assert_script_run("git clone https://github.com/k8snetworkplumbingwg/whereabouts && cd whereabouts", 600);
+    assert_script_run("git clone --depth 1 https://github.com/k8snetworkplumbingwg/whereabouts && cd whereabouts", 600);
```

Comment:
The changes in agent and server will impact also sles kubevirt test, right? If yes, maybe good to add a verification for 15sp6?

Reply:
Yes, will run it.